apps/odo/lib/odo.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Odo do
+  @moduledoc """
+  Convenience functions for the Odo application.
+  """
+
+  def event_stream_instance() do
```

Comment:
It'd be nice if this was closer to our convention in other apps (just `instance()`) but  I understand if that feels out of scope for this card.

Reply:
this was more or less a backwards compatibility issue. We created reaper's `instance()` function (or equivalent) to return the atom `:reaper_brook` but now we have other non-Brook use cases for an `instance`-type function and we didn't want to break the existing view state by renaming that instance return value when we have a live system using it.